README.md

```diff
@@ -4,23 +4,19 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
 # Installation
 This addon uses the [Dota 2 Addon Manager](https://github.com/chrisinajar/dota2-addon-manager).
 
-For detailed instructions, check [here](docs/install.md)
+For installation instructions click [here](docs/install.md).
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
-Check out the Discord!
+Check out our [Discord](https://discord.gg/E6f6TCe)!
```

Comment:
That's also link, to this server, just created by me, also permanent/general/oaa but generated from my account

Tenure - review (t9 from phone)